civet.dev/cheatsheet.md

```diff
@@ -315,3 +315,39 @@ floor # / 2
 ```
 
 </div>
+
+## ASCII Symbols
+
+Here is a list of ASCII symbols and their various contextual meanings in Civet.
+(There are also more [Unicode symbols](/reference#unicode-operators).)
+
+| Symbol | Meanings |
+|--------|----------|
+| space | Function call |
```

Comment:
Actually, if we're listing space here, it can also be a TS type argument (`Set number` = `Set<number>`)